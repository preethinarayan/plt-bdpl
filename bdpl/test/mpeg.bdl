//
// This is an MPEG Transport Stream Processor
//
struct TransportPacket{
	byte sync satisfies {sync == 0x47} then {} else {print("Error in processing packet: "); print($#a-1);};
}[1] tp;

file "x:\test\mpeg.ts" inputFile ;
read(inputFile,tp);
