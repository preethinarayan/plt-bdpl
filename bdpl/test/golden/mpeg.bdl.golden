Packet 20 is scrambled
111 packets found
